trimmer_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..10]
+  end
```

Comment:
- [ ] The `correct_name` method should return a maximum of 10 characters. If the name is longer, it should trim the word. To achieve this, you should have `super[0..9]` instead of `super[0..10]`. Please modify your code accordingly 👍